odl/util/numerics.py

```diff
@@ -742,13 +748,15 @@ def _apply_padding(lhs_arr, rhs_arr, offset, pad_mode, direction):
             bdry_slc_l[axis] = left_slc
             bdry_slc_r = list(working_slc)
             bdry_slc_r[axis] = right_slc
+            bdry_slc_l, bdry_slc_r = map(tuple, [bdry_slc_l, bdry_slc_r])
```

Comment:
Just write one per line

```
bdry_slc_l = tuple(bdry_slc_l)
bdry_slc_r = tuple(bdry_slc_r)
```

Slight DRY violation, but it makes the code much more readable and maintainable for non-pythonistas.

Reply:
I'll do the cases with 2, but the ones with 4 just feel wrong to put in 4 lines. If the functional version is not great, I can also do list comprehension:
```python
# Old
lhs_slc_l, rhs_slc_l, lhs_slc_r, rhs_slc_r = map(
    tuple, [lhs_slc_l, rhs_slc_l, lhs_slc_r, rhs_slc_r])
# New
lhs_slc_l, rhs_slc_l, lhs_slc_r, rhs_slc_r = [
    tuple(lst) for lst in [lhs_slc_l, rhs_slc_l, lhs_slc_r, rhs_slc_r]
]
```
Or I just write a little inline helper
```python
def to_tuple(*args):
    return tuple(tuple(arg) for arg in args)
```

Reply:
For 3 or more, I'm fine with the `map` approach, its just that for 2 it's barely any reduction.